src/Twig/Node/Element.php

```diff
@@ -74,7 +74,7 @@ public function compile(Compiler $compiler)
     {
         $compiler->addDebugInfo($this);
 
-        $compiler->raw('echo $context[\'_view\']->element(');
+        $compiler->raw('echo $context[\'this\']->element(');
```

Comment:
The var name is hard-coded here. This means if someone changes the var name it will stop working isn't it?

Reply:
That is very true. I can extend the custom variable unit test to include element and cell. Hopefully we remember if adding another one.